java-checks/src/test/files/checks/UselessExtendsCheck.java

```diff
@@ -0,0 +1,12 @@
+public class UselessExtendsCheck { // Compliant
+
+  public class Class1 { // Compliant
+  }
+
+  public class Class2 extends Object { // Noncompliant
+  }
+
+}
+
+public class Class3 extends java.lang.Object { // Noncompliant
```

Comment:
Lack a testcase where the symbol in the extend clause is not known (just use a dummy name)